common/model-views.cpp

```diff
@@ -683,7 +683,16 @@ namespace rs2
                                 float new_value;
                                 if (!string_to_int(buff, new_value))
```

Comment:
Also consider to refactor that one with std::stoi

Reply:
same explanation as above